config/webpack.config.js

```diff
@@ -18,18 +20,18 @@ let cfg = {
 
   entry: {
     main: ['./src/main.ts'],
-    'assets/init': ['./src/frontend/javascript/init.js'],
-    'assets/sankey': ['./src/frontend/javascript/sankey.js'],
+    init: {import: './src/frontend/javascript/init.js', filename: 'assets/init.js'},
+    sankey: {import: './src/frontend/javascript/sankey.js', filename: 'assets/sankey.js'}
   },
 
   output: {
-    path: path.resolve(__dirname, '..', 'dist'),
-    filename: '[name].js',
     publicPath: '/',
+    path: path.resolve(__dirname, '..', 'dist'),
```

Comment:
@kr8n3r just flagging this, I had a surprisingly difficult time trying to get the manifest plugin working yesterday, due to useless errors. In the end re-adding this path fixed it. Tests pass, looks OK, sound fine to you?